fontforgeexe/fontview.c

```diff
@@ -1029,25 +1030,193 @@ char *GetPostScriptFontName(char *dir, int mult) {
 return( temp );
 }
 
+struct gfc_data {
+    int done;
+    int ret;
+    GGadget *gfc;
+    GGadget *ignore_invalid;
+    SplineFont *sf;
+    EncMap *map;
+};
+
+static int e_h(GWindow gw, GEvent *event) {
```

Comment:
This function looks good to me, but do you mind calling it something a tad more descriptive, like `fv_e_h` or `feat_e_h`? That's the convention used elsewhere.